src/Files.App/Services/NetworkService.cs

```diff
@@ -5,55 +5,47 @@
 using System.Text;
 using Vanara.InteropServices;
 using Vanara.PInvoke;
+using Vanara.Windows.Shell;
+using static SkiaSharp.HarfBuzz.SKShaper;
 using static Vanara.PInvoke.AdvApi32;
 using static Vanara.PInvoke.Mpr;
 
 namespace Files.App.Services
 {
-	public sealed class NetworkDrivesService : ObservableObject, INetworkDrivesService
+	public sealed class NetworkService : ObservableObject, INetworkService
 	{
 		private ICommonDialogService CommonDialogService { get; } = Ioc.Default.GetRequiredService<ICommonDialogService>();
 
 		private readonly static string guid = "::{f02c1a0d-be21-4350-88b0-7367fc96ef3c}";
 
 
-		private ObservableCollection<ILocatableFolder> _Drives;
+		private ObservableCollection<ILocatableFolder> _Computers;
 		/// <inheritdoc/>
-		public ObservableCollection<ILocatableFolder> Drives
+		public ObservableCollection<ILocatableFolder> Computers
 		{
-			get => _Drives;
-			private set => SetProperty(ref _Drives, value);
+			get => _Computers;
+			private set => SetProperty(ref _Computers, value);
+		}
+
+		private ObservableCollection<ILocatableFolder> _Shortcuts;
+		/// <inheritdoc/>
+		public ObservableCollection<ILocatableFolder> Shortcuts
+		{
+			get => _Shortcuts;
+			private set => SetProperty(ref _Shortcuts, value);
 		}
 
 		/// <summary>
-		/// Initializes an instance of <see cref="NetworkDrivesService"/>.
+		/// Initializes an instance of <see cref="NetworkService"/>.
 		/// </summary>
-		public NetworkDrivesService()
+		public NetworkService()
 		{
-			_Drives = [];
-
-			var networkItem = new DriveItem()
-			{
-				DeviceID = "network-folder",
-				Text = "Network".GetLocalizedResource(),
-				Path = Constants.UserEnvironmentPaths.NetworkFolderPath,
-				Type = DriveType.Network,
-				ItemType = NavigationControlItemType.Drive,
-			};
-
-			networkItem.MenuOptions = new ContextMenuOptions()
-			{
-				IsLocationItem = true,
-				ShowEjectDevice = networkItem.IsRemovable,
-				ShowShellItems = true,
-				ShowProperties = true,
-			};
-			lock (_Drives)
-				_Drives.Add(networkItem);
+			_Computers = [];
```

Comment:
Can you both move to the property definitions?

Reply:
Done.